options.go

```diff
@@ -377,6 +379,41 @@ func (o *CommitOptions) Validate(r *Repository) error {
 	return nil
 }
 
+var (
+	ErrMissingName    = errors.New("name field is required")
+	ErrMissingTagger  = errors.New("tagger field is required")
+	ErrMissingMessage = errors.New("message field is required")
+)
+
+// TagObjectOptions describes how a tag object should be created.
+type TagObjectOptions struct {
```

Comment:
`s/TagObjectOptions/CreateTagOptions/` to match current convention of `<Function>Options`